README.md

```diff
@@ -25,53 +25,53 @@ Add the tasks to your build definition.
-* _(Optional)_. Set the `custom version` (in a `<M>.<m>.<p>` semver format) if needed for example : 'v1.22+3' or '1.17.0'.
+* Select the `channel`: `stable` (default), `beta`, or `dev`.
+* Select the `version` of the SDK to install:  `latest` (default), `custom`. If `custom` is specified, a `customVersion` must be set.
+* _(Optional)_. Set the `customVersion` (in a `<M>.<m>.<p>` semver format) if needed.
 
 ### Build
 
 ![](images/step_build.png)
 
-Build the given mobile application project. You must call the `Flutter Install` task, set a `FlutterToolPath` environment variable, or use the optional Flutter SDK Path task entry that points to your `flutter/bin` folder before execution. All the application bundles are created into the `build/outputs` folder of your project.
-
-* Select the `project source directory` (that contains to `pubspec.yaml` file).
-* Select the `target` platform: `Android (default)`, `iOS`, or `All` for both.
-* _(Optional)_. Set `flutter sdk path` if using a local agent with a pre-installed Flutter SDK, can specify the path to utilize it.  Otherwise use Flutter Install.
-* _(Optional)_. Set `package name` (like `1.2.3`) that will override the manifest's one.
-* _(Optional)_. Set `package number` (like `12`) that will override the manifest's one.
-* _(Optional)_. Set `build flavour` (like `development`) to specify a build flavour.  Must match Android Gradle flavor definition or XCode scheme.
-* _(Optional)_. Set `verbose` if you wish to get detailed verbose log output for diagnoses purposes.
-* _(Optional)_. Set `debug` if you wish to override the default release mode for the build.
-* _(Optional)_. Set `dart-define` compile-time variables. Example : "Some_Var=Some_val --dart-define=Some_Var2=Val"
-* _(Optional)_. Set `extraArgs` if you want to pass more official/custom command arguments. Example : "--no-tree-shake-icons --publish-to-play"
-* __(Android)__._(Optional)_. Set `platform` for the Android target: `android-arm (default)`, `android-arm64`.
-* __(Android)__._(Optional)_. Set the build mode `split-per-abi` to compile the code into an APK per target ABI.  Otherwise the build will result in a single APK.
-* __(iOS)__._(Optional)_. Set `platform` for the iOS target: `device (default)`, `simulator`.
-* __(iOS)__._(Optional)_. Codesign the application bundle (only available on device builds, and activated by default). **Warning: you must install a valid certificate before build with the `Install an Apple Certificate`task**
+Build the given mobile application project. You must call the `Flutter Install` task or use the optional `flutterDirectory` task input that points to your `flutter/bin` folder before execution. All application bundles are created in the `build/outputs` folder of your project.
+
+* Select the `projectDirectory` that contains the `pubspec.yaml` file.
+* Select the `target` platform. Options are: `apk` (default), `aab`, `ios`, `web` or `all` for both Android and iOS, but without Web.
```

Comment:
Hi, thank you for your PR. Could you add the new changes here

"options": {
                "allweb": "All (with 'web')", // this and
                "all": "All (except 'web')", // this too
                "ios": "iOS",
                "apk": "Android (apk)",
                "aab": "Android (aab)",
                "web": "Web"
            }

Reply:
Thanks for checking. Updated the PR.

As a side note: I think that `all` and `allweb` are quite confusing labels. 
What do you think about changing to `mobile` instead of `all` and `all` instead of `allweb`?

Reply:
Actually, they will change either way in respect to #4 for now I think your PR is good for current version.